content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Propagating the Trace Context
+title: Trace Context Propagation
```

Comment:
For some reason, going to this page puts us in the OTEL part of the docs, rather than APM. Can we fix that?
![image](https://github.com/DataDog/documentation/assets/12128670/e139c31f-b469-44e3-863c-6ff883021229)

Reply:
I've attempted to fix this issue in commit e02c2fe202d29f8c88ea831534a45daae200daf5

Reply:
This is because the menu file is pointing to the same page from two locations (and it prefers the one higher up, aka OTel). I've fixed this here b9bdc32d12d65efe939583b833e70fc74baf0563